async_simple/CMakeLists.txt

```diff
@@ -30,15 +30,15 @@ if(UTHREAD)
   list(APPEND SRCS ${uthread_asm_src})
 endif()
 
-add_library(async_simple_static STATIC ${SRCS})
-add_library(async_simple SHARED ${SRCS})
+add_library(async_simple STATIC ${SRCS})
```

Comment:
Is this change necessary?

Reply:
If use the original config, will generate async_imple.dll on windows, and all the test cases depends on the async_simple.lib and compile error on windows, so here don't use set_target_properties. Explicit generate static and shared lib.

Reply:
Now the behavior changes more. Currently, the shared library wouldn't get installed. Also the name changes. Both is not good. I would suggest to wrap line 38 for windows only.